packages/insomnia-app/app/ui/components/request-pane.js

```diff
@@ -319,6 +342,16 @@ class RequestPane extends React.PureComponent<Props> {
                 )}
               </button>
             </Tab>
+            {pluginTabs.map((p: PluginTab, i: number) => (
+              <Tab tabIndex="-1" key={i}>
```

Comment:
[We should avoid index as keys](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) with React in general, but especially here.

I think using a combination of the plugin name and tab label should be sufficient to ensure keys are unique.

```suggestion
              <Tab tabIndex="-1" key={`${p.plugin.name}::${p.label}`}>
```